gradio/processing_utils.py

```diff
@@ -187,11 +187,14 @@ def audio_from_file(filename, crop_min=0, crop_max=100):
     try:
         audio = AudioSegment.from_file(filename)
     except FileNotFoundError as e:
-        error_message = str(e)
-        if "ffprobe" in error_message:
+        isfile = os.path.isfile(filename)
+        print(f"Cannot load audio from file: `{'ffprobe' if isfile else filename}` not found, {e}.")
```

Comment:
Thanks for the change @FelixDombek !

Why not `raise ValueError(<text you added in this pr>)` ? 

Happy to merge once this is resolved.

Reply:
The added benefit of adding this text to the exception is that if the demo is launched with `show_error=True`, this helpful error message will show up in the client. Making it much more actionable.

<img width="1384" alt="image" src="https://user-images.githubusercontent.com/41651716/209951048-687711be-0465-42e9-a3e8-8aa73a592e98.png">

Reply:
The `ValueError` doc says it's used when "an argument has the right type but the wrong value", but ffprobe isn't an argument. I think it would be nicer to pass on the FileNotFoundError with the additional info, let me try that ...

Reply:
Sounds good!

Reply:
The Python docs raise a RuntimeError in this case (https://docs.python.org/3/tutorial/errors.html#exception-chaining), so that's what I went with now.